databases/week1/database.sql

```diff
@@ -0,0 +1,70 @@
+--1.Find out how many tasks are in the task table
+
+SELECT COUNT(*)
+FROM HYF.task;
```

Comment:
curious why you use `HYF.task` here and not just `task` like in the other exercises?